source/ed/ed2nav.cpp

```diff
@@ -165,6 +164,41 @@ struct FindAdminWithCities {
     }
 };
 
+bool rename_old_file(const std::string& output_filename, std::string& backup_output_filename) {
+    auto logger = log4cplus::Logger::getInstance("log");
```

Comment:
you can give a more specific name for the logger 
`auto logger = log4cplus::Logger::getInstance("ed2nav::rename_old_file");`